### PR TITLE
Upgrade to storybook 8

### PR DESCRIPTION
**Pull request by KaranAtTeamForm**

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+    "@storybook/core-events",
+    "@chromatic-com/storybook",
+    "@storybook/preset-create-react-app",
+    "@storybook/addon-webpack5-compiler-babel"
   ],
+
   framework: {
     name: "@storybook/react-webpack5",
-    options: {}
+    options: { fastRefresh: true }
```
Comment (wtfiwtz) [inline .storybook/main.js:20]: Does this work? https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#frameworkoptionsfastrefresh-for-webpack5-based-projects-removed

File: .storybook/preview.js [hunk 2]
```diff
@@ -109,3 +119,4 @@ export const parameters = {
   viewMode: "docs",
   chromatic: { pauseAnimationAtEnd: true }
 };
+// export const tags = ["autodocs"];
```
Comment (wtfiwtz) [inline .storybook/preview.js:122]: remove?

**Review by wtfiwtz — APPROVED** [review]
🏆 Thanks Karan!

File: .storybook/main.js (hunk 1 above)
Comment (KaranAtTeamForm) [reply .storybook/main.js]: You are right, does not work.

File: .storybook/preview.js (hunk 2 above)
Comment (KaranAtTeamForm) [reply .storybook/preview.js]: Removed.

**Review by wtfiwtz — APPROVED** [review]

**Review by wtfiwtz — APPROVED** [review]

**Pull request merged**